src/strategies/cToken/index.ts

```diff
@@ -0,0 +1,94 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Arr00';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'account',
+        type: 'address'
+      }
+    ],
+    name: 'balanceOf',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'account',
+        type: 'address'
+      }
+    ],
+    name: 'borrowBalanceStored',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  }
+
+];
+
+export async function strategy(network, provider, addresses, options, snapshot) {
+  let blockTag = typeof snapshot === 'number' ? (snapshot) : 'latest';
+ 
+
+  const [balanceNowResponse, borrowResponse]  = await Promise.all([
```

Comment:
It's better if we can reduce the number of request being sent. I recommend to merge the 2 multicall requests into a single multicall request.